src/whylogs/core/metrics/model_metrics.py

```diff
@@ -10,22 +11,35 @@ class ModelMetrics:
 
     Attributes:
         confusion_matrix (ConfusionMatrix): ConfusionMatrix which keeps it track of counts with NumberTracker
+        regression_metrics (RegressionMetrics): Regression Metrics keeps track of a common regression metrics in case the targets are continous.
     """
 
-    def __init__(self, confusion_matrix: ConfusionMatrix = None):
-        if confusion_matrix is None:
-            confusion_matrix = ConfusionMatrix()
+    def __init__(self, confusion_matrix: ConfusionMatrix = ConfusionMatrix(),
```

Comment:
Don't initialize object here as part of function parameters

Reply:
fixed in 8dc266e